build.sbt

```diff
@@ -32,11 +34,48 @@ lazy val allSettings = Seq(
         <name>Ólafur Páll Geirsson</name>
         <url>https://geirsson.com</url>
       </developer>
+      <developer>
+        <id>jonas</id>
+        <name>Jonas Fonseca</name>
+        <url>https://github.com/jonas</url>
+      </developer>
     </developers>
 )
 
 lazy val example = project
 
+lazy val protobufSettings = Seq(
+  PB.targets.in(Compile) := Seq(
```

Comment:
I find `a.in(b)` clearer, esp. when you do stuff like `a.in(b, c, d).value.map(...)`.